game/static/game/css/landing.css

```
@@ -146,26 +207,62 @@ body {
+        to bottom, 
+        #ffffff 20%, 
+        #d4af37 50%, 
+        #8a6d1a 80%
+    );
+    -webkit-background-clip: text;
+    background-clip: text;
+    filter: drop-shadow(0 15px 30px rgba(0, 0, 0, 0.5));
+    animation: titleEntrance 1.2s cubic-bezier(0.2, 0, 0.2, 1);
+}
+
+@keyframes titleEntrance {
```

Stylelint (17.11.0)

[error] Expected keyframe name "titleEntrance" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)